Analysis/Utkscan/core/source/UtkUnpacker.cpp

```diff
@@ -60,6 +60,9 @@ void UtkUnpacker::ProcessRawEvent(ScanInterface *addr_/*=NULL*/) {
         PrintProcessingTimeInformation(systemStartTime, times(&systemTimes),
             GetEventStartTime(), eventCounter);
 
+    //set eventWidth from Globals (loaded from config file)
+    eventWidth = Globals::get()->GetEventLengthInTicks();
```

Comment:
You mean the first event is already build when UtkUnpacker::InitializeDriver() called? I will try initializing [there](https://github.com/pixie16/paass/blob/master/Scan/utkscan/core/source/UtkScanInterface.cpp#L64) then.